spec/jobs/socure_shadow_mode_proofing_job_spec.rb

```diff
@@ -349,22 +349,59 @@
       job.build_applicant(encrypted_arguments:, user_email:)
     end
 
-    it 'builds an applicant structure that looks right' do
-      expect(build_applicant).to eql(
-        {
-          first_name: 'FAKEY',
-          last_name: 'MCFAKERSON',
-          address1: '1 FAKE RD',
-          address2: nil,
-          city: 'GREAT FALLS',
-          state: 'MT',
-          zipcode: '59010-1234',
-          phone: '12025551212',
-          dob: '1938-10-06',
-          ssn: '900-66-1234',
-          email: user.email,
-        },
-      )
+    let(:expected_attributes) do
+      {
+        first_name: 'FAKEY',
+        last_name: 'MCFAKERSON',
+        address1: '1 FAKE RD',
+        address2: nil,
+        city: 'GREAT FALLS',
+        state: 'MT',
+        zipcode: '59010-1234',
+        dob: '1938-10-06',
+        ssn: '900-66-1234',
+        email: user.email,
+      }
+    end
+
+    context 'when the user has a phone directly passed in' do
+      # existing test
+      it 'builds an applicant structure that looks right' do
+        expect(build_applicant).to eql(
+          expected_attributes.merge(phone: '12025551212'),
+        )
+      end
+    end
+
+    context 'when the user only has an MFA phone' do
+      let(:applicant_pii_no_phone) do
+        Idp::Constants::MOCK_IDV_APPLICANT_WITH_SSN.merge(
```

Comment:
Arguably `MOCK_IDV_APPLICANT_WITH_SSN` is what we should be using by default, because it appears there is currently no way for phone to get in. Moving from `MOCK_IDV_APPLICANT_WITH_PHONE` to `MOCK_IDV_APPLICANT` also loses SSN which we don't want.